systemd/ignition-firstboot-complete.service

```diff
@@ -7,9 +7,11 @@ RequiresMountsFor=/boot
 [Service]
 Type=oneshot
 RemainAfterExit=yes
+# Remount /boot temporarily writable; https://github.com/ostreedev/ostree/issues/1265
+MountFlags=slave
 # We should have only run if GRUB detected this file. Fail if we are unable to
 # remove it, rather than risking rerunning Ignition at next boot.
-ExecStart=/bin/rm /boot/ignition.firstboot
+ExecStart=/bin/sh -c 'mount -o remount,rw /boot && rm /boot/ignition.firstboot'
```

Comment:
Should we change this back to ro after the `rm /boot/ignition.firstboot` or does it not matter because everything gets unmounted/remounted again soon?

Reply:
The `MountFlags=slave` creates a new mount namespace; from the point of view of everything else, `/boot` stays readonly.  We only have a transient writable mount for the lifetime of the unit.

Reply:
ahh. any chance you could add that exact sentence under the comment on line 10. That's a great summary.

Reply:
Sure, done.